Automattic/course
# Add support for Learning Mode

Resolves https://github.com/Automattic/course/issues/46 , https://github.com/Automattic/course/issues/47

## Changes proposed in this pull request

- Updated theme.json and styles to make free and paid Learning Mode templates match the design

## Testing instructions

- Checkout this brunch of the Sensei Plugin https://github.com/Automattic/sensei/pull/6068
- Have the latest trunk of Sensei Pro
- Make sure when you go to sensei settings, you get all 4 templates for LM
- Make sure to enable LM
- Create a course with several modules, containing several lessons each
- Make sure to add course videos in some of those lessons, also don't add any videos in some of them
- Add quizzes in some of the lessons so that the "Take Quiz" button appears at the bottom
- Now check the course as a student
- Make sure to switch to all templates and for each of the templates, check at least one lesson with a video and one without
- Make sure the designs look as expected, with acceptably close font size and spacings to the design and overall structure matches

## Screenshots

**Default**
![Screen Shot 2022-11-19 at 4 21 50 AM](https://user-images.githubusercontent.com/6820724/202813194-ef5f23fc-cfdd-4218-b6a0-8eee77c94e9d.png)

**Video**
![Screen Shot 2022-11-19 at 4 20 55 AM](https://user-images.githubusercontent.com/6820724/202813250-c049194c-a8bb-47d5-9df9-bf72f7ca28a4.png)

**Modern**
![Screen Shot 2022-11-19 at 4 21 24 AM](https://user-images.githubusercontent.com/6820724/202813295-ebff7190-e497-4be6-a0ef-b7402759c432.png)

**Large Video**
![Screen Shot 2022-11-19 at 4 23 27 AM](https://user-images.githubusercontent.com/6820724/202813324-d3f99e86-949f-4697-a63a-01686d013efa.png)
